src/sentry/api/endpoints/organization_details.py

```diff
@@ -606,6 +610,10 @@ def put(self, request: Request, organization) -> Response:
         """
         from sentry import features
 
+        # This param will be used to determine if we should include feature flags in the response
+        include_feature_flags = request.GET.get("include_feature_flags", "0") != "0"
+        (include_feature_flags)
```

Comment:
is this a typo?

Reply:
i currently don't use the flag because i need to first update calls to the endpoint after this. to prevent linting errors, i added that temporarily, probably should put a comment about it.

Reply:
ahhh got it! I think you could also just do ` _ = some_call()`

Reply:
I would add a TODO here to avoid temporary confusion

Reply:
+1